src/BrazeMessageComponent.tsx

```diff
@@ -34,6 +36,7 @@ const COMPONENT_MAPPINGS: ComponentMapping = {
     [APP_BANNER_NAME]: AppBanner,
     [SPECIAL_EDITION_BANNER_NAME]: SpecialEditionBanner,
     [THE_GUARDIAN_IN_2020_BANNER_NAME]: TheGuardianIn2020Banner,
+    [US_NEWSLETTER_EPIC_NAME]: USNewsletterEpic,
```

Comment:
I think we've agreed to pull this out into a separate factory component specific to `EndOfArticle` which would have it's own mappings. Something like `BrazeEndOfArticleComponent`. I think it's fine to do that in a separate PR though.